clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreBufferService.java

```diff
@@ -319,6 +322,7 @@ public void stopInner() throws Exception {
       this.executorService.shutdownNow();
       this.executorService.awaitTermination(10, TimeUnit.SECONDS);
     }
+    isStarted = false;
```

Comment:
Should we move this to the beginning of stopInner?

Reply:
Sure